pkg/imageverifycache/client.go

```diff
@@ -75,19 +101,29 @@ func (c *cache) Set(ctx context.Context, policy kyvernov1.PolicyInterface, ruleN
 	if !c.isCacheEnabled {
 		return false, nil
 	}
-	c.logger.Info("Successfully set cache", "policy", policy.GetName(), "ruleName", ruleName, "imageRef", imageRef)
+	key := generateKey(policy, ruleName, imageRef)
+
+	stored := c.Cache.SetWithTTL(key, nil, 1, c.ttl)
```

Comment:
don't we need to `cache.Wait()` ?

Reply:
@eddycharly I have asked in ristretto community about it. 

>  it’s used commonly in tests to wait until all asynchronous operations on the cache are completed. Not normally needed, but seems to be sort of a manual override to Ristretto’s normal operation.

I think there is no need to add cache.wait() because ristretto has used this function internally.